migtests/scripts/functions.sh

```diff
@@ -684,19 +684,24 @@ EOF
 assess_migration() {
 	args="--export-dir ${EXPORT_DIR}
 		--source-db-type ${SOURCE_DB_TYPE}
-		--source-db-host ${SOURCE_DB_HOST}
 		--source-db-port ${SOURCE_DB_PORT}
 		--source-db-user ${SOURCE_DB_USER}
 		--source-db-password ${SOURCE_DB_PASSWORD}
 		--source-db-name ${SOURCE_DB_NAME}
 		--send-diagnostics=false --yes
 		--start-clean t
-		--iops-capture-interval 0
+		--iops-capture-interval 1
 	"
 	if [ "${SOURCE_DB_SCHEMA}" != "" ]
 	then
 		args="${args} --source-db-schema ${SOURCE_DB_SCHEMA}"
 	fi
+	if [ "${SOURCE_DB_ORACLE_TNS_ALIAS}" != "" ]
```

Comment:
Question: If TNS_ALIAS is provided, does SCHEMA will still be required?

Reply:
Ideally schema shouldn't be required. But there's other dependencies in our test structure for it. So it's kept that way for now. 
We can restructure it later if need be.